frontend/pages/colorMatch.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+
+  <div>
+    <h1>Color Match</h1>
+    <!-- waiting queue -->
+    <div v-if="state.gameState === 0">Warte auf weitere Spieler...</div>
+    <div v-else-if="state.gameState === 2">Es läuft bereits ein Spiel. Bitte habe einen Moment Geduld...</div>
+<!--    <div v-else-if="state.gameState === 3">Spiel ist offen.</div>-->
+    <div v-else-if="state.gameState === 1">
+      <div v-if="!state.solved">
+        <p>
+          Willkommen beim Spiel Color Match. Euer Ziel ist es, beide Lampen in den gleichen Farbe leuchten zu lassen.
+        </p>
+        <p>Du hast die Farbe <b>{{ colorDict[state.color] }}</b> erhalten.</p>
+
+        <!-- slider -->
+        <vue-slider style="position: relative; top: 24px" tooltip='none' :min="0" :max="255" v-model="currentTry" height="4" :dotSize=30></vue-slider>
+        <div v-if="state.color === 'red'">
+          <progress style="position: relative" class="progress is-danger" :value="currentTry" max="255">15%</progress>
+        </div>
+        <div v-if="state.color === 'green'">
+          <progress style="position: relative" class="progress is-success" :value="currentTry" max="255">15%</progress>
+        </div>
+        <div v-if="state.color === 'blue'">
+          <progress style="position: relative" class="progress is-info" :value="currentTry" max="255">15%</progress>
+        </div>
```

Comment:
Verrschönerung: Es gibt in Vue ":class" (https://vuejs.org/guide/essentials/class-and-style.html#binding-html-classes)